2.0.0/update.py

```diff
@@ -0,0 +1,227 @@
+    'E': 'disgust',
+    'A': 'fear',
+    'F': 'happiness',
+    'T': 'sadness',
+    'N': 'neutral',
+}
+transcription_mapping = {
+    'a01': 'Der Lappen liegt auf dem Eisschrank.',
+    'a02': 'Das will sie am Mittwoch abgeben.',
+    'a04': 'Heute abend könnte ich es ihm sagen.',
+    'a05': 'Das schwarze Stück Papier befindet sich da oben neben dem '
+           'Holzstück.',
+    'a07': 'In sieben Stunden wird es soweit sein.',
+    'b01': 'Was sind denn das für Tüten, die da unter dem Tisch '
+           'stehen.',
+    'b02': 'Sie haben es gerade hochgetragen und jetzt gehen sie '
+           'wieder runter.',
+    'b03': 'An den Wochenenden bin ich jetzt immer nach Hause '
+           'gefahren und habe Agnes besucht.',
+    'b09': 'Ich will das eben wegbringen und dann mit Karl was '
+           'trinken gehen.',
+    'b10': 'Die wird auf dem Platz sein, wo wir sie immer hinlegen.',
+}
+scheme_mapping = {
+    "emotion":"emotion",
+    "emotion.confidence":"confidence",
+    "emotion.naturalness":"naturalness",
+    "speaker":"speaker",
+    "duration":"duration",
+    "transcription":"transcription",
+    "gender":"gender",
+    "language":"language",
+}
+def convert_german_float(x):
+    x = float(x.replace(",", "."))
+    return x/100
+# Prepare functions for getting information from file names
+def parse_names(names, from_i, to_i, is_number=False, mapping=None):
+    for name in names:
+        key = name[from_i:to_i]
+        if is_number:
+            key = int(key)
+        yield mapping[key] if mapping else key
+
+# Same constants
+
+name = 'emodb'
+previous_version = '1.4.1'
+build_dir = './build'
+build_dir = audeer.mkdir(build_dir)
+folder_audio = "audio"
+folder_laryngo = "laryngo"
+
+# Preparation
+
+# Remove the "xx" in the name from all new files
+file_list = os.listdir("audio")
+for folder in [folder_audio, folder_laryngo]:
+    for f in file_list:
+        f_new = f.replace("xx.wav", ".wav")
+        os.rename(os.path.join(folder, f), os.path.join(folder, f_new))
+
+# Parse the list with label agreements and naturalness
+annotations = pd.read_csv("listener_judgements.txt", sep="\t", index_col="sample")
+# Align the index with database index
+annotations = annotations.set_index(annotations.index.to_series()\
+                                    .map(lambda x: "wav/"+x))
+list_cols = ["recognized", "natural"]
+for col in list_cols:
+    annotations[col] = annotations[col].map(lambda x: convert_german_float(x))
+annotations = annotations.rename(columns={"recognized":"emotion.confidence",\
+                                          "natural":"emotion.naturalness"})
+
+# Get the new files
+# file_list_all = sorted(
+#     [f for f in os.listdir(audio_folder)]
+# )
+file_list_all = os.listdir(folder_audio)
+
+# Get the old files
+db = audb.load_to(
+    build_dir,
+    name,
+    version=previous_version,
+    num_workers=8,
+    cache_root=None,
+    verbose=True,
+)
+file_list_exist = db["files"].get().index.values
```

Comment:
To address all files in a dataset, please use:

```suggestion
file_list_exist = db.files.values
```

Which is most likely identical here to `db["files"].index`, but it does not have to be.